digiwf-engine/digiwf-engine-service/src/main/resources/prozesse/example/dms-integration/show-case/forms/dokument-updaten.schema.json

```
@@ -67,6 +67,32 @@
                   "required"
                 ]
               },
+              "contentCoos": {
+                "fieldType": "dms-input",
+                "title": "Schriftstück Coos",
```

Der Titel hat mich erst verwirrt, da im Formular keine COOs sondern Titel und Dateityp angezeigt werden.
```suggestion
                "title": "Schriftstücke",
```